### PR TITLE
ability to run on macOS using a docker

### PR DESCRIPTION
**Pull request by belkone**
I added the option to run using Docker. Personally, it's a faster way for me than using Vagrant or virtual machines on macOS;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,22 @@ go build
 
 sudo packer build boards/odroid-u3/archlinuxarm.json
 ```
+## Run in Docker
+This method is primarily for macOS users where is no native way to use qemu-user-static.
+Dockerfile is modeled on [packer-builder-arm-image](https://github.com/solo-io/packer-builder-arm-image).
+#### Build docker image:
```
Comment (mkaczanowski) [inline README.md]: '###' instead of '####'?

**Comment by mkaczanowski** [comment]
Does the 'Dockerfile' etc. have to be in the root directory? can you move files to /docker?

**Comment by belkone** [comment]
Sure, I will do it tonight ;)

File: README.md (hunk 1 above)
Comment (belkone) [reply README.md]: I can change it, but it's not a typo. It's a subsection of Docker's section. Do you want to have it changed?

Comment (mkaczanowski) [reply README.md]: I am not following :) If
> ## Run in Docker

is a parent section of:
> #### Build docker image

Then it should be one "#" more, thus 3* "#" = "###", right?

Comment (belkone) [reply README.md]: oh of course you are right! I wrote it late and when I read your comment I did not verify it :)

**Comment by mkaczanowski** [comment]
Thanks!

**Pull request merged**